src/coreclr/jit/emitarm64.cpp

```diff
@@ -21017,6 +21036,11 @@ void emitter::emitDispInsHelp(
             emitDispSveReg(id->idReg3(), INS_OPTS_SCALABLE_D, false); // mmmmm
             break;
 
```

Comment:
Could you please add a comment above this case describing the encoding format? e.g. `// <Pd>.H, <Pn>.B`

Reply:
+1